dashboard/app/models/script_level.rb

```diff
@@ -77,6 +77,10 @@ def has_another_level_to_go_to?
     end
   end
 
+  def final_level?
+    !has_another_level_to_go_to?
+  end
+
```

Comment:
nit: (perhaps for another PR) Does it make sense to deprecate `has_another_level_to_go_to?` in favor of `final_level?`? It doesn't make much sense to me to keep them both, especially given their limited usage.